### ISSUE TITLE
Offline Mode

### ISSUE THREAD

**Issue by Olofsn** [EvotecIT/PSWriteHTML#115]
I don't know how many people need this, but I only work on servers that are not connected to the internet. I would be nice to have the option to run it all offline.

**Comment by PrzemyslawKlys** [comment]
What specifically is not working for you?

New-HTML has two switches:
- UseCssLinks
- UseJavaScriptLinks

If you use those it requires internet to work. If you don't most of the functionality should work offline. 

The only difference is for Out-HtmlView which defaults to internet use. This could probably be changed. 

What actually happens when you skip those switches is that the content is put inline.


Most of the stuff that's missing can be probably fixed quite easily. It's stored in https://github.com/EvotecIT/PSWriteHTML/blob/master/Private/Parameters.Configuration.ps1

Basically all code that has JSLink and doesn't have JS or CSSLink and doesn't have Css requires fixing. 

For example:

![image](https://user-images.githubusercontent.com/15063294/75369076-00973d00-58c3-11ea-8c17-4ebdef751c52.png)

So anything specific doesn't work?

**Comment by Olofsn** [comment]
Hi,

thanks for the quick answer. I actually testet it with Out-HtmlView.
But If the other functions don’t need internet, I’m happy with that.

Thank you again for the quick reply and your work!

Olofsn

**Comment by PrzemyslawKlys** [comment]
This is now live. Out-HtmlViee is by default Offline. -Online switch makes it use online.
